Source/engine/random.cpp

```diff
@@ -51,4 +51,9 @@ int32_t GenerateRnd(int32_t v)
 	return AdvanceRndSeed() % v;
 }
 
+bool flipCoin()
```

Comment:
I wasn't able to come up with a better name, but being a global the casing is wrong:
```suggestion
bool FlipCoin()
```